stable/rabbitmq/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: rabbitmq
-version: 4.3.2
+version: 4.3.3
```

As you are adding a new feature, following [semver versioning](https://semver.org/) you should update the minor version instead.

```suggestion
version: 4.4.0
```